### DISCUSSION TITLE
Does PyTorch/NumPy randomization bug affects MONAI?

### DISCUSSION THREAD

**Discussion by dzenanz** [Project-MONAI/MONAI#2002] [General]
Does the bug described here
https://tanelp.github.io/posts/a-bug-that-plagues-thousands-of-open-source-ml-projects/
affect augmentation or anything else in MONAI?

**Comment by wyli** [comment]
to my knowledge that's addressed in monai https://github.com/Project-MONAI/MONAI/issues/398
https://github.com/Project-MONAI/MONAI/blob/2707407fed8c78ccb1c18d1e994a68580457219e/monai/data/utils.py#L390-L397
